parity/cli.rs

```diff
@@ -168,7 +167,7 @@ Legacy Options:
   --nodekey KEY            Equivalent to --node-key KEY.
   --nodiscover             Equivalent to --no-discovery.
   -j --jsonrpc             Does nothing; JSON-RPC is on by default now.
-  -w --webapp              Does nothing; web app server is on by default now.
+  -d --dapps               Does nothing; dapps server is on by default now.
```

Comment:
don't change the legacy options - they're only there to make old code work and come with a deprecated warning.